docs/software/remote_desktop.md

```diff
@@ -33,80 +34,212 @@
+
+
+## Remot3.it - (Weaved) Access your device over the internet
+
+Remot3.it allows you to easily access your DietPi device over the internet.
+
+![DietPi remote desktop software Remot3.it](../assets/images/dietpi-software-remotedesktop-remot3it.png){: style="width:600px"}
+
+Weaved works by connecting you to a specific TCP port on your device, all of which can be customized during first run setup.
+
+Examples of TCP ports for Remot3.it:
+
+- SSH port 22. Open a remote terminal to your device.
+- Transmission port 9091. Monitor your BitTorrent downloads.
+- Webserver port 80. Access your internal webserver and/or websites.
+
+=== "First Run Setup"
+
+    The first run setup is executed once `dietpi-software` has completed the installation and the system is rebooted. Run the following command. Follow the onscreen instructions to setup and link an account:
+
+    - `connectd_installer`
+
+    Once your account is created and linked to this system, you can select a port for Remot3.it to enable remote access.
 
+=== "Access your device"
 
-### VNC Server - Shared desktop
+    Sign into your Remot3.it account to access your devices remotely:
 
-Edit dietpi.txt and enable VNC server to run in _shared desktop_ mode:
+    - URL = <https://www.remot3.it>
 
-```
-nano /boot/dietpi.txt
-```
+## VirtualHere - Share physically attached USB devices from your SBC over the network
 
-and change to value 1 next line
-```
-SOFTWARE_VNCSERVER_SHARE_DESKTOP=1
-```
+The VirtualHere package is used to share physically attached USB devices from your SBC over the network to other systems.
 
-A running desktop is required in this mode, please set desktop to start from boot:
-```
-dietpi-autostart
-```
+![DietPi remote desktop software VirtualHere](../assets/images/dietpi-software-remotedesktop-virtualhere.png){: style="width:600px"}
 
-### VNC Server - Resolution settings
+Download the client for your PC from:
 
-Here it's an example on how to run VNC server on screen :1, creating a new desktop with 1280x720 resolution:
+- <https://www.virtualhere.com/usb_client_software>
 
-```
-SOFTWARE_VNCSERVER_WIDTH=1280
-SOFTWARE_VNCSERVER_HEIGHT=720
-SOFTWARE_VNCSERVER_DEPTH=32
-SOFTWARE_VNCSERVER_DISPLAY_INDEX=1
-```
+Once installed, available VirtualHere devices will be shown in the client user interface and can be used on the client PC.
 
-Restart service:
-```
-systemctl restart vncserver.service
-```
+!!! warning "USB Storage WARNING"
+    - As per <https://github.com/MichaIng/DietPi/issues/852#issuecomment-292781475> it it highly recommended that you do not install VirtualHere if your DietPi user data is stored on a USB drive.
+    - VirtualHere does not take into account mounted drives when selecting them for remote use. This is potentially dangerous for any mounted drive that is in use and may cause data loss.
+    - Do not use drives on the client that are mounted on the SBC.  
+      Unmount the drive before hand in `dietpi-drive_manager`.
```

Comment:
Could be verified if it's still true. However no harm keeping this information present for now, I'll have a look/test tomorrow.

Reply:
Ok